src/backend/sqlite/foreign_key.rs

```diff
@@ -21,9 +21,9 @@ impl ForeignKeyBuilder for SqliteQueryBuilder {
         &self,
         create: &ForeignKeyCreateStatement,
         sql: &mut SqlWriter,
-        inside_table_creation: bool,
+        mode: Mode,
     ) {
-        if !inside_table_creation {
+        if mode != Mode::Creation {
```

Comment:
I think you can remove this `if`

Reply:
Okay, I will remove it, but if a user tries to modify existing foreign constrain the database will return an error. Is it a good way to handle this situation ?

Reply:
Ah, I get it! Sorry, I thought we panic anyway...

Reply:
so, do I remove it or not ?

Reply:
No, it is ok.